gamestonk_terminal/menu.py

```diff
@@ -1,17 +1,21 @@
+import logging
 import os
 
 from matplotlib import pyplot
 from prompt_toolkit import PromptSession
 from prompt_toolkit.eventloop.inputhook import set_eventloop_with_inputhook
 from prompt_toolkit.history import FileHistory
 
+logger = logging.getLogger(__name__)
+
 
 def inputhook(inputhook_context):
     while not inputhook_context.input_is_ready():
         try:
             pyplot.pause(0.1)
         # pylint: disable=unused-variable
-        except Exception:  # noqa: F841
+        except Exception as exp:  # noqa: F841
+            logger.exception("%s", type(exp).__name__)
```

Comment:
f-string here

Reply:
Same comment logging should be done more explicitly per pylint - https://pycodequ.al/docs/pylint-messages/w1203-logging-fstring-interpolation.html